torch/csrc/jit/codegen/cuda/executor.cpp

```diff
@@ -274,6 +274,47 @@ at::Tensor inferAndAlloc(
   }
 }
 
+at::Tensor inferAndAlloc(
```

Comment:
Is the old `inferAndAlloc` going to be removed? I see it's still used for allocating outputs.

Reply:
Old inferAndAlloc is still used for static allocations, where it didn't make sense to memoize expression.

Reply:
The two versions look quite similar. Can they be merged?